manifests/namespace-install.yaml

```diff
@@ -254,42 +254,42 @@ data:
         - version: latest
           natsImage: nats:2.9.16
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: /nats-server
         - version: 2.8.1
           natsImage: nats:2.8.1
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: /nats-server
         - version: 2.8.1-alpine
           natsImage: nats:2.8.1-alpine
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: nats-server
         - version: 2.8.2
           natsImage: nats:2.8.2
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: /nats-server
         - version: 2.8.2-alpine
           natsImage: nats:2.8.2-alpine
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: nats-server
         - version: 2.9.1
           natsImage: nats:2.9.1
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: /nats-server
         - version: 2.9.12
           natsImage: nats:2.9.12
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
+          configReloaderImage: natsio/nats-server-config-reloader:0.14.1
           startCommand: /nats-server
         - version: 2.9.16
           natsImage: nats:2.9.16
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
-          configReloaderImage: natsio/nats-server-config-reloader:0.7.0
```

Comment:
Instead updating old versions, maybe add the latest nats version with the new config reloader image? This config is always supposed to be maintained by the users.

Reply:
sure, `latest` makes sense

Reply:
What I meant was, still add the latest nats image `2.10.x`, and also update the other images like `config-reloader` with new version, but only for `2.10.x` config, and remain the histories unchanged.

Reply:
thanks, updated.